independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/project/quarkus/base/src/main/resources/META-INF/resources/index.tpl.qute.html

```diff
@@ -0,0 +1,288 @@
+{#if generate-native and quarkus.platform.version.compareVersionTo("3.9") < 0}
```

Comment:
what does "generate-native" mean?

Reply:
oups that's a bad copy and paste good catch!!!

Reply:
fixed!

Reply:
FYI I tested locally and it worked as expected, but that was just luck..